src/runtime/shared_mem.cc

```diff
@@ -76,26 +76,28 @@ void *SharedMemory::CreateNew(size_t size) {
   CHECK_NE(fd, -1) << "fail to open " << name << ": " << strerror(errno);
   // Shared memory cannot be deleted if the process exits abnormally.
   AddResource(name, std::shared_ptr<Resource>(new SharedMemoryResource(name)));
-  auto res = ftruncate(fd, size);
+  auto res = ftruncate(fd, sz);
   CHECK_NE(res, -1)
       << "Failed to truncate the file. " << strerror(errno);
-  ptr = mmap(NULL, size, PROT_READ|PROT_WRITE, MAP_SHARED, fd, 0);
+  ptr = mmap(NULL, sz, PROT_READ|PROT_WRITE, MAP_SHARED, fd, 0);
   CHECK_NE(ptr, MAP_FAILED)
       << "Failed to map shared memory. mmap failed with error " << strerror(errno);
+  this->size = sz;
```

Comment:
This is exactly why we should follow google coding style for naming members. Would you mind changing it to `size_`?